### PR TITLE
MGMT-6027 - update documentation for deploying Assisted Service v ia …

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…Operator with mirror configuration

            Add exmaple for the config map that defines registries.conf and certificate

### CODE REVIEW

File: docs/operator.md [hunk 1]
```diff
@@ -168,7 +168,37 @@ In case user wants to create an installation that uses mirror registry, the foll
...
+       location = "quay.io/ocpmetal"
+       mirror-by-digest-only = false
+    
+       [[registry.mirror]]
+       location = "bm-cluster-1-hyper.e2e.bos.redhat.com:5000/ocpmetal"
+EOF
+```
+
+   Note1: ConfigMap should be installed in the same namespace as the assisted-service-operator (ie. `assisted-installer`).
+   Note2: registry.conf supplied should use "mirror-by-digest-only = false" mode
```
Comment (nmagnezi) [inline docs/operator.md:200]: it renders Note2 as a continued line of Note1.
If you'll have a line space between them, it should fix it

**Review by nmagnezi — COMMENTED** [review]
I can't say I'm familiar with those areas, so maybe someone who does should also look. Yet a couple of minor formatting comments.
1. The commit-msg topic was probably too long, so Github messed it up. Please fix it.
2. commented inline

File: docs/operator.md (hunk 1 above)
Comment (yevgeny-shnaidman) [reply docs/operator.md:201]: fixed

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**